src/protocols/secure_channel/PASESession.cpp

```diff
@@ -284,19 +287,14 @@ CHIP_ERROR PASESession::WaitForPairing(uint32_t mySetUpPINCode, uint32_t pbkdf2I
 
 CHIP_ERROR PASESession::WaitForPairing(const PASEVerifier & verifier, uint16_t myKeyId, SessionEstablishmentDelegate * delegate)
 {
-    CHIP_ERROR err = WaitForPairing(0, kSpake2p_Iteration_Count, reinterpret_cast<const unsigned char *>(kSpake2pKeyExchangeSalt),
-                                    strlen(kSpake2pKeyExchangeSalt), myKeyId, delegate);
-    SuccessOrExit(err);
+    ReturnErrorOnFailure(WaitForPairing(0, kSpake2p_Iteration_Count,
+                                        reinterpret_cast<const unsigned char *>(kSpake2pKeyExchangeSalt),
```

Comment:
nit: should "unsigned char" here be uint8_t for consistency with the rest of the codebase? 

Realise this code is not really changed at this level, so optional change for sure if it even makes sense.

Reply:
`WaitForPairing` is exposed to the rest of the code as an API, and it currently takes `unsigned char` as the argument. We should be able to change it to `uint8_t`, but maybe better to do this in a separate PR.